tests/installation/installation_overview.pm

```diff
@@ -69,6 +69,20 @@ sub run {
         $self->deal_with_dependency_issues;
         assert_screen "inst-xen-pattern" if get_var('XEN');
         ensure_ssh_unblocked;
+        # Check the systemd target, see poo#45020
+        if (get_var('DESKTOP')) {
+            my $target;
+            if (check_var('DESKTOP', 'textmode')) {
+                $target = "multi-user";
+            }
+            else {
+                $target = "graphical";
+            }
+            select_console 'install-shell';
```

Comment:
I'm not 100% if that will work on hyperv, for example. Complexity here is that this module is executed almost everywhere. `logs_from_installation_system` is another module where we switch between consoles and it's skipped for quite some scenarios. Could you please crosscheck?

Reply:
I'd like to , but it won't run with openqa-clone-custom-git-refspec. I get this error:
```
Stop-VM : Hyper-V was unable to find a virtual machine with name 
"openQA-SUT-1".
```

Reply:
https://openqa.suse.de/tests/2547445

Reply:
Mm, I thought I had tested it on s390, but it seems not. That is not working here: https://openqa.suse.de/tests/2547444#step/installation_overview/24

Reply:
Finally a VR actually ran, and confirmed this does **not** work on hyperv. But, it works on s390.

Reply:
Which is not bad, actually. I would expect that it will be worse =)